internal/host/validator.go

```diff
@@ -703,12 +706,12 @@ func (v *validator) hasSufficientNetworkLatencyRequirementForRole(c *validationC
 	if len(c.host.Connectivity) == 0 {
 		return ValidationFailure
 	}
-	status, _, _ := v.validateNetworkLatencyForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts)
+	status, _, _ := v.validateNetworkLatencyForRole(c.host, c.clusterHostRequirements, c.cluster.Hosts, c.cluster.OpenshiftVersion)
 	return status
 }
 
-func (v *validator) validateNetworkLatencyForRole(host *models.Host, clusterRoleReqs *models.ClusterHostRequirements, hosts []*models.Host) (ValidationStatus, []string, error) {
-	var connectivityReport *models.ConnectivityReport
+func (v *validator) validateNetworkLatencyForRole(host *models.Host, clusterRoleReqs *models.ClusterHostRequirements, hosts []*models.Host, openshiftVersion string) (ValidationStatus, []string, error) {
```

Comment:
It was proposed with @gamli75 and @avishayt as part of the metrics labels. On a second thought, the latency and packet loss are intrinsic values of the hosts, not something that will change based on the openshift version. If so, I'll remove it and leave only the `sourceRole` and `targetRole`.

Reply:
Just wanted to know.  If required then leave it.

Reply:
You have a point anyway. I actually don't see much value in adding the openshift version here since it won't change the outcome of the latency and packet loss. It might make more sense to include which virt provider (if any) is being used.

Reply:
@jordigilh don't remove it - let's leave it as is

Reply:
Ok, I'll keep it then.